applications/ConvectionDiffusionApplication/tests/test_symbolic_convection_diffusion_explicit_element/materials_steady_state_explicit_solution.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I'd name this json file with `_materials`. What comes to my mind with `_solution` is the reference values to check to.